packages/superinput/src/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface IPanelProps {
+  valid: boolean | null
+  text: string
+}
+
+export enum SI_TYPES {
+  TEMPLATE = 0,
+  EXPRESSION = 1,
+  BOOL = 2
+}
+
+export interface ISiProps {
+  value?: string
+  onChange?: (newValue: string) => any
+  placeholder?: string
+  type?: SI_TYPES
+  globs?: any
+  noGlobsEvalMsg?: string
+  autoFocus?: boolean
+}
```

Comment:
Minor comment about coding conventions. We don't usually prefix our interfaces with `I`. Also enums should be CamelCase